cypher_connected_people.go

```diff
@@ -31,41 +30,38 @@ func (pcw CypherDriver) ConnectedPeople(uuid string, fromDateEpoch int64, toDate
 	WITH p2.uuid as uuid, p2.prefLabel as prefLabel, cm as count, content as contentList
 	RETURN prefLabel, uuid, count, contentList
 	ORDER BY count DESC LIMIT {limit}`
-	//thing := Thing{}
+
 	query := &neoism.CypherQuery{
 		Statement: statement,
 		Parameters: neoism.Props{
 			"uuid":               uuid,
 			"fromDate":           fromDateEpoch,
 			"toDate":             toDateEpoch,
 			"minimumConnections": minimumConnections,
-			"limit":              limit,
+			"limit":              resultLimit,
 			"contentLimit":       contentLimit,
 		},
 		Result: &results,
 	}
-	err = pcw.db.Cypher(query)
-	if err != nil {
+	log.Debugf("Query %v", query)
+
+	if err = cd.conn.CypherBatch([]*neoism.CypherQuery{query}); err != nil || len(results) == 0 {
 		log.Errorf(`Error finding people with more than %v connections to person with uuid %v
-      between %v and %v with the following statement: %v  Error: %v`, limit, uuid, fromDateEpoch, toDateEpoch, query.Statement, err)
-		return []ConnectedPerson{}, false, fmt.Errorf("Error finding people with more than %v connections to person with uuid %v between %v and %v with the following statement: %v  Error: %v", limit, uuid, fromDateEpoch, toDateEpoch, query.Statement, err)
-	}
-	log.Infof("CypherResult connectedPeople was (fromDate=%v, toDate=%v): %+v", fromDateEpoch, toDateEpoch, results)
-	if (len(results)) == 0 {
-		return []ConnectedPerson{}, false, nil
+      between %v and %v with the following statement: %v  Error: %v`, resultLimit, uuid, fromDateEpoch, toDateEpoch, query.Statement, err)
+		return []ConnectedPerson{}, false, err
```

Comment:
Shouldn't there be the case when returning false for not found, but no error? I saw that when you call this, you check for that also, but I can't see this return case here.